documentation/smg/12_deploying_releases_to_GitHub.md

```diff
@@ -0,0 +1,89 @@
+# Deploying releases to GitHub
+
+## ci-release-deploy Repository
+
+A repository
+[ci-release-deploy](https://github.com/pace-neutrons/ci-release-deploy)
+has been created to hold the code that implements the deployment pipeline.
+This decision was made so that code is not duplicated between Herbert and Horace
+and to provide fast checkout times for the deploy pipelines.
+
+## Release Pipelines
+
+The release pipelines in ANVIL's Jenkins server should be used to create release
```

Comment:
It would he helpful to note that for Horace builds this takes a Herbert Branch/Tag that will be bundled into the Horace release.

That's implicit in "this is the same as `master`" but worth emphasising